astro/src/content/blog/test-saml-configuration.mdx

```diff
@@ -0,0 +1,115 @@
+
+### Retrieve The Configuration Values From The IAMShowcase Website
+
+The IAMShowcase site provides a metadata file that some IdPs can import. You can get the XML from the IAMShowcase site by clicking on [Get Metadata](https://sptest.iamshowcase.com/instructions#start). FusionAuth does not support importing of the metadata file. To see the values used, view the XML and manually enter the information in the SAML tab of the application in FusionAuth.
+
+Example of metadata provided by IAMShowcase:
+```
+<md:EntityDescriptor xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata" xmlns:ds="http://www.w3.org/2000/09/xmldsig#" entityID="IAMShowcase" validUntil="2025-12-09T09:13:31.006Z">
+<md:SPSSODescriptor AuthnRequestsSigned="false" WantAssertionsSigned="true" protocolSupportEnumeration="urn:oasis:names:tc:SAML:2.0:protocol">
+<md:NameIDFormat>urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified</md:NameIDFormat>
+<md:NameIDFormat>urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress</md:NameIDFormat>
+<md:AssertionConsumerService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" Location="https://sptest.iamshowcase.com/acs" index="0" isDefault="true"/>
+</md:SPSSODescriptor>
+</md:EntityDescriptor>
+```
+
+### Configure The IAMShowcase Example Application For SAML
+
+Select <Breadcrumb>Applications</Breadcrumb> and <InlineUIElement>Edit</InlineUIElement> the IAMShowcase Example application in FusionAuth. Select the <Breadcrumb>SAML</Breadcrumb> tab of the application and configure it with the following values.
+
+<InlineField>Enabled</InlineField> `True`
+
+<InlineField>Issuer</InlineField> `IAMShowcase`
+This is the entityID from the EntityDescriptor node in the matadata provided by IAMShowcase.
+```
+<md:EntityDescriptor xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata" xmlns:ds="http://www.w3.org/2000/09/xmldsig#" entityID="IAMShowcase" validUntil="2025-12-09T09:13:31.006Z">
+```
+
+<InlineField>Authorized Redirect URLs</InlineField> `https://sptest.iamshowcase.com/acs`
+This is the Location from the AssertionConsumerService node in the matadata provided by IAMShowcase.
+```
+<md:AssertionConsumerService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" Location="https://sptest.iamshowcase.com/acs" index="0" isDefault="true"/>
+```
+<InlineField>Signing key</InlineField> `Auto generate a new key on save...`
+
+Leave all other defaults the same, and click the blue disk icon button to save it.
+
+![IAMShowcase Example Application SAML configuration.](/img/blogs/test-saml-configuration/fusionauth-app-saml-settings.png)
+
+### Add The User To The IAMShowcase Example Application 
+
+Select <InlineUIElement>Users</InlineUIElement>. Choose a user you wnat to test the SAML configuration with. Select <InlineUIElement>Manage</InlineUIElement>. <InlineUIElement>Add registration</InlineUIElement> for the IAMShowcase Example application for the user.
```

Comment:
I'd suggest reading the blog post aloud, as some of the sentences are a bit awkward. I know it is a pain, but you'll catch things.

Reply:
```suggestion
Select <InlineUIElement>Users</InlineUIElement>. Choose a user you want to test the SAML log in process with. Select <InlineUIElement>Manage</InlineUIElement> and then click the <InlineUIElement>Add registration</InlineUIElement> button for the IAMShowcase Example application. Click the save icon and you have registered the user for the application.
```